src/Azure.Functions.Cli/Kubernetes/KubernetesHelpers.cs

```diff
@@ -528,5 +627,71 @@ private static ConfigMapV1 GetConfigMap(string name, string @namespace, IDiction
                 Data = secrets
             };
         }
+
+        public static ServiceAccountV1 GetServiceAccount(string name, string @namespace)
+        {
+            return new ServiceAccountV1
+            {
+                ApiVersion = "v1",
+                Kind = "ServiceAccount",
+                Metadata = new ObjectMetadataV1
+                {
+                    Name = name,
+                    Namespace = @namespace
+                }
+            };
+        }
+
+        public static RoleV1 GetRole(string name, string @namespace)
```

Comment:
rename to reflect role permissions specified below since this is not a generic "Role" creator.